man/systemd.exec.xml

```diff
@@ -1653,6 +1653,26 @@ StateDirectory=aaa/bbb ccc</programlisting>
         <xi:include href="version-info.xml" xpointer="v234"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>StateDirectoryQuota=</varname></term>
+        <term><varname>CacheDirectoryQuota=</varname></term>
+        <term><varname>LogsDirectoryQuota=</varname></term>
+
+        <listitem><para>Specifies the storage limits for the directories specified in <varname>StateDirectory=</varname>,
+        <varname>CacheDirectory=</varname>, or <varname>LogsDirectory=</varname> respectively.</para>
+
+        <para>The storage quota is defined in terms of disk blocks and inodes, as per
+        <ulink url="https://man7.org/linux/man-pages/man2/quotactl.2.html">quotactl</ulink>. Takes an absolute size limit in bytes. If the value
+        is suffixed with K, M, G or T, the specified size is parsed as Kilobytes, Megabytes, Gigabytes, or Terabytes (with the base 1024), respectively.
+        If an absolute size limit is specified, only the block quota is set (rounded down to the nearest block). Alternatively, a percentage value may be specified,
+        which applies the same percent quota to both blocks and inodes. Defaults to <constant>0%</constant>, in which case no storage limits will be set.</para>
```

Comment:
i really don't like that 0% is supposed to be "no limits". if anything 100% would mean that, and 0% "no writes allowed".

Reply:
but actually i'd prefer if we have special value for turning this off, i.e. if the literal string "off" is specfied we should read that as "no quota enforcement".